spec/services/comparison_service_spec.rb

```diff
@@ -14,12 +14,18 @@
 
   describe 'refresh_all_comparisons' do
     it 'recovers from releasecop comparison failures' do
-      project # ensure project and org are created
-      allow_any_instance_of(ComparisonService).to receive(:refresh_comparisons).and_raise(ZeroDivisionError)
+      allow_any_instance_of(ComparisonService).to receive(:refresh_comparisons)
+        .and_raise(ZeroDivisionError)
       expect { ComparisonService.refresh_all_comparisons }.not_to raise_error
     end
   end
 
+  describe 'refresh_comparisons_for_organizations' do
+    it 'works' do
+      expect { ComparisonService.refresh_comparisons_for_organization(org) }.not_to raise_error
+    end
```

Comment:
This one actually calls the action cable broadcast and throws the exact same error that we saw on papertrail.